src/background/bgPopupHandler.ts

```diff
@@ -80,3 +80,34 @@ export const openPopup = async (
 export const removePopup = (id: string) => delete popups[id];
 
 export const getPopup = (id: string): IPopupConfigNoActions => popups[id];
+
+export const setSessionExpires = async (expires: number) => {
+  // @ts-expect-error session storage is not defined
+  await browser.storage.session.set({ sessionExpires: expires });
+};
+
+export const getSession = async () => {
+  try {
+    // @ts-expect-error session storage is not defined
```

Comment:
To avoid multiple `// @ts-expect-error` I suggest to define the session as const and reuse it:
```
const storageSession = (browser.storage as any).session;
```